beacon-chain/p2p/broadcaster.go

```diff
@@ -124,15 +126,13 @@ func (s *Service) internalBroadcastAttestation(ctx context.Context, subnet uint6
 		if err := func() error {
 			s.subnetLocker(subnet).Lock()
 			defer s.subnetLocker(subnet).Unlock()
-			ok, err := s.FindPeersWithSubnet(ctx, attestationToTopic(subnet, forkDigest), subnet, 1)
-			if err != nil {
-				return err
-			}
-			if ok {
-				savedAttestationBroadcasts.Inc()
-				return nil
+
+			if err := s.FindPeersWithSubnets(ctx, AttestationSubnetTopicFormat, forkDigest, minimumPeersPerSubnetForBroadcast, map[uint64]bool{subnet: true}); err != nil {
```

Comment:
what is the benefit of passing the minimum peers as an argument on the function instead of something set inside s. or as a field param

Reply:
also what benefits are there for passing in map[uint64]bool{subnet: true} ? that seems like some constant, it doesn't seem like its used in this function either why wouldn't we just set it inside find peerswithsubnets?